Possible async deadlock fixed (#1302)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

* **[Feature]** Support ARM64 architecture.
* **[Breaking change]** The minimum supported Window 10 version is now `10.0.16299.0`.

#### WPF

> * **[Fix]** The possible deadlock on `App Center` start was fixed (touch events were not fired after `App Center` start from `OnStartup`).

Reviewer: ```suggestion
* **[Fix]** Fix a deadlock when starting AppCenter that was preventing touch events from being fired.
```
